app/models/setting/katello.rb

```diff
@@ -26,7 +26,8 @@ def self.load_defaults
         self.set('pulp_client_key', N_("Path for ssl key used for pulp server auth"), "/etc/pki/katello/private/pulp-client.key"),
         self.set('pulp_client_cert', N_("Path for ssl cert used for pulp server auth"), "/etc/pki/katello/certs/pulp-client.crt"),
         self.set('remote_execution_by_default', N_("If set to true, use the remote execution over katello-agent for remote actions"), false),
-        self.set('use_pulp_oauth', N_("use oauth authentication for pulp instead of the default cert based authentication"), false)
+        self.set('use_pulp_oauth', N_("use oauth authentication for pulp instead of the default cert based authentication"), false),
+        self.set('enable_lazy_sync', N_("enable pulp's lazy sync feature"), true)
```

Comment:
I'm a bit hesitant to use the term lazy sync here since it doesn't appear anywhere else in Katello or pulp. I believe the term pulp uses is "deferred downloads".